### PR TITLE
Update composition.md

### PR DESCRIPTION
**Pull request by braghettos**
Fixed tabbing at row 446

### Description of your changes

Fixes #

I have:

- [ ] Read and followed Crossplane's [contribution process].
- [ ] Run `make reviewable test` to ensure this PR is ready for review.

### How has this code been tested

[contribution process]: https://git.io/fj2m9

### CODE REVIEW

File: docs/introduction/composition.md [hunk 1]
```diff
@@ -819,4 +819,4 @@ known issues and proposed improvements.
 [Infrastructure Composition Provisioning]: composition-provisioning.png
 [composition related issues]: https://github.com/crossplane/crossplane/labels/composition
 [#1481]: https://github.com/crossplane/crossplane/issues/1481
-[#1909]: https://github.com/crossplane/crossplane/issues/1909
\ No newline at end of file
+[#1909]: https://github.com/crossplane/crossplane/issues/1909
```
Comment (hasheddan) [inline docs/introduction/composition.md:822]: This looks like a newline modification, potentially unintentional?

**Review by hasheddan — COMMENTED** [review]
Thanks @braghettos! Could you run the following to sign your commit?

```
git commit --amend --signoff
git push -f
```

File: docs/introduction/composition.md (hunk 1 above)
Comment (braghettos) [reply docs/introduction/composition.md:822]: I don't know why it adds this row!

Comment (hasheddan) [reply docs/introduction/composition.md]: It is likely just a setting in your editor, no worries if it hangs around :) I actually couldn't tell if it was adding or removing a newline, we generally prefer to have a newline so maybe it is actually fixing a missing one :+1:

**Pull request closed without merging**